macros/src/main/scala/gestalt/macros/DefMacros.scala

```diff
@@ -20,6 +20,26 @@ object plusObject {
         toolbox.Lit(null)
     }
   }
+
+  inline def varargs(tped: Int*): Int = meta {
+    val q"$items: $_" = tped
```

Comment:
`Ascribe` is used here to extract out of the `Typed` block that for some reason is wrapped around varargs: both `SeqLiteral`s and all others.